Chapter 6/plugins/config.js

```diff
@@ -6,7 +6,6 @@ const fastifyEnv = require('@fastify/env')
 module.exports = fp(async function configLoader (fastify, opts) {
   await fastify.register(fastifyEnv, {
     confKey: 'secrets',
-    data: opts.configData,
```

Comment:
just as memo: this is necessary for injecting config settings during the test process.

It will be added in the next chapters